R/escape.R

```diff
@@ -69,14 +69,13 @@ escape.factor <- function(x, parens = NA, collapse = ", ", con = NULL) {
 
 #' @export
 escape.Date <- function(x, parens = NA, collapse = ", ", con = NULL) {
-  x <- as.character(x)
-  escape.character(x, parens = parens, collapse = collapse, con = con)
+  x = as.character(x)
```

Comment:
I think you should delete this line; `sql_escape_date()` should be in charge of converting from a date to a character

Reply:
Moved to coverting bit to sql_escape_date.